impl/maven-impl/src/main/java/org/apache/maven/impl/DefaultDependencyResolver.java

```diff
@@ -67,6 +67,21 @@
 @Named
 @Singleton
 public class DefaultDependencyResolver implements DependencyResolver {
+    /**
+     * Cache of information about the modules contained in a path element.
+     * This cache is created when first needed. It may be never created.
+     *
+     * <p><b>TODO:</b> This field should not be in this class, because the cache should be global to the session.
+     * This field exists here only temporarily, until clarified where to store session-wide caches.</p>
+     *
+     * @see moduleCache()
```

Comment:
```suggestion
     * @see #moduleCache()
```
<img width="857" alt="image" src="https://github.com/user-attachments/assets/bcc5a0dc-9147-4a79-80f7-796d97bba8cb" />

cs should fail this build.

Reply:
Side note: no need for CheckStyle, as `javac` can do this verification itself with the `-Xdoclint` option. It is not enabled in Maven maybe for historical reasons. Or maybe because we want to be able to use a `@see`  tag to another module without causing build failure. The latter is the kind of problem that "multi-modules project" resolves. But this is for another day, maybe far in the future.

Reply:
Javadoc link fixed.